apiary.apib

```diff
@@ -375,6 +375,7 @@ We list all backwards-compatible additions here. These are currently available i
 - We added `term` to the filters on `workTypes.list`.
 - We added `started_on` to `timeTracking.update` and `timeTracking.add`.
 - We added the `quotations.delete` endpoint.
+- timeTracking.list filters started_before, started_after, ended_before and ended_after will now include the datetime passed in the results.
```

Comment:
I would also word this differently:

> We changed the behaviour of the `started_before`, `started_after`, `ended_before`, and `ended_after` filters on `timeTracking.list` to be inclusive. So "started after midnight" will include time tracking that started exactly on midnight.

But that is a suggestion. I'm sure it could be explained better 😄

Reply:
This does make it more clear 👍  changed it